tests/assets/bundles.yaml

```diff
@@ -22,4 +22,4 @@ install:
   bundles:
     - rootfs_path: /var/lib/extensions/kubo
       targets:
-        - container://ttl.sh/bundles-test:24h
\ No newline at end of file
+        - container://quay.io/kairos/ci-temp-images:bundles-test
```

Comment:
Any job might have built this image (since it always has the same name). So if multiple are run in parallel, they will be overwriting this same image. The image will be the same no matter where it's built but this might not be the best idea. It was like that already so no need to fix it here.

Reply:
yep indeed, when I checked this I found it weird. No idea how to fix this. It helps that the image and contents seem to be always the same and havent changed in years lol,  but what should we do? a fixed image? something that doesnt expire? I have no workaround for this :(